datacommons/stat_vars.py

```diff
@@ -203,17 +203,33 @@ def get_stat_all(places, stat_vars):
       }
     """
     url = utils._API_ROOT + utils._API_ENDPOINTS['get_stat_all']
-    req_json = {'stat_vars': stat_vars, 'places': places}
-
-    # Send the request
-    res_json = utils._send_request(url, req_json=req_json, use_payload=False)
-
-    if 'placeData' not in res_json:
-        raise ValueError('No data in response.')
-
-    # Unnest the REST response for keys that have single-element values.
-    place_statvar_series = collections.defaultdict(dict)
-    for place_dcid, place in res_json['placeData'].items():
-        for stat_var_dcid, stat_var in place['statVarData'].items():
-            place_statvar_series[place_dcid][stat_var_dcid] = stat_var
-    return dict(place_statvar_series)
+    places = list(places)
+    # Ceil hack to get # of batches.
+    batches = -(-len(places) // utils._QUERY_BATCH_SIZE)
+    res = {}
+    no_data = True
```

Comment:
it might be better to use has_data here?

Reply:
Let me know if this comment still holds after my response here: https://github.com/datacommonsorg/api-python/pull/155#discussion_r486546024

it would be a logic reversal with no behavioral changes.

Reply:
here i actually mean a wording change, use has_data as oppose to no_data for variable naming, just readability nit :)